src/views/examples/Profile.js

```diff
@@ -161,173 +108,179 @@ const Profile = () => {
                           >
-                            Country
+                            Github Profile Link
                           </label>
                           <Input
                             className="form-control-alternative"
-                            defaultValue="United States"
-                            id="input-country"
-                            placeholder="Country"
-                            type="text"
+                            defaultValue={props.githubLink}
+                            id="github-profile-link"
+                            // placeholder={props.githubLink}
+                            type="url"
                           />
                         </FormGroup>
                       </Col>
-                      <Col lg="4">
+                    </Row>
+                    <Row>
+                      <Col md="12">
                         <FormGroup>
                           <label
                             className="form-control-label"
-                            htmlFor="input-country"
+                            htmlFor="Linkedin-Profile-Link"
                           >
-                            Postal code
+                            Linkedin Profile Link
                           </label>
                           <Input
                             className="form-control-alternative"
-                            id="input-postal-code"
-                            placeholder="Postal code"
-                            type="number"
+                            defaultValue={props.linkedinLink}
+                            id="linkedin-profile-link"
+                            // placeholder={props.linkedinLink}
+                            type="url"
                           />
                         </FormGroup>
                       </Col>
                     </Row>
                   </div>
-                  <hr className="my-4" />
-                  {/* Description */}
-                  <h6 className="heading-small text-muted mb-4">About me</h6>
-                  <div className="pl-lg-4">
-                    <FormGroup>
-                      <label>About Me</label>
-                      <Input
-                        className="form-control-alternative"
-                        placeholder="A few words about you ..."
-                        rows="4"
-                        defaultValue="A beautiful Dashboard for Bootstrap 4. It is Free and
-                        Open Source."
-                        type="textarea"
-                      />
-                    </FormGroup>
-                  </div>
                 </Form>
               </CardBody>
             </Card>
           </Col>
         </Row>
+        <div className="text-center mt-3">
+            <Button
+              color="primary"
+              href="#pablo"
+              onClick={(e) =>{ console.log("save-clicked"); e.preventDefault();}}
```

Comment:
Define a function that calls when someone clicks this button and as of now that function should console current values of Resume Link, Github link, etc.

Reply:
Maybe create a dummy API call too in the function. The request mode would be `POST`.